### PR TITLE
Modifications In set_trustline.py

### PR DESCRIPTION
**Pull request by samitabar**
Checks If You Have Sufficient Balance for creating trustlines

### CODE REVIEW

File: src/airdrops/set_trustline.py [hunk 1]
```diff
@@ -7,10 +7,11 @@
 
 from random import randint
 
+import xrpy
```
Comment (amiwrpremium) [inline src/airdrops/set_trustline.py]: Import what you use only

File: src/airdrops/set_trustline.py [hunk 2]
```diff
@@ -98,32 +99,37 @@ def mass_trust_line(path_to_csv: str, skip_already_set: bool, currency: str, val
...
+                        print(colored(text=f'Traceback: {traceback.format_exc()}', color='red'))
                     continue
-
-            except Exception as e:
-                report.add_failed()
-                print(colored(text=f'Error: {e}', color='red'))
-                if debug or __debug:
-                    print(colored(text=f'Traceback: {traceback.format_exc()}', color='red'))
-                continue
+                else:
```
Comment (amiwrpremium) [inline src/airdrops/set_trustline.py]: Wrong Indent

File: src/airdrops/set_trustline.py [hunk 3]
```diff
@@ -98,32 +99,37 @@ def mass_trust_line(path_to_csv: str, skip_already_set: bool, currency: str, val
...
-                    print(colored(
-                        text=f'Status: Success',
-                        color='green'
-                    ))
-
-                    sleep_time = randint(min_sleep_time, max_sleep_time)
-                    print(colored(text=f"Sleeping for {sleep_time} seconds. zZz...", color='blue'))
-                    time.sleep(sleep_time)
-                else:
+            balance = xrpy.get_account_info(XRP_MAIN_CLIENT, wallet.classic_address).result.get("account_data").get("Balance")
```
Comment (amiwrpremium) [inline src/airdrops/set_trustline.py]: No exception handling here

File: Readme.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 <h3>Requirements:</h3>
 <ul>
 <li>Install <a href="https://www.python.org">Python 3.8+</a></li>
-<li>Download Source: <code>git clone https://github.com/amiwrpremium/airdrops.git</code></li>
+<li>Download Source: <code>git clone https://github.com/samitabar/airdrops.git</code></li>
```
Comment (amiwrpremium) [inline Readme.md:5]: Don't change project url

**Review by amiwrpremium — CHANGES_REQUESTED** [review]
Some changes are required before merging

**Comment by samitabar** [comment]
Fixed The Bugs Mentioned Above

File: src/airdrops/set_trustline.py [hunk 5]
```diff
@@ -99,8 +98,17 @@ def mass_trust_line(path_to_csv: str, skip_already_set: bool, currency: str, val
...
+            try:
+                balance = get_account_info(XRP_MAIN_CLIENT, wallet.classic_address).result.get("account_data").get("Balance")
+                xrp_balance = int(balance)/1000000
+            except Exception as e:
+                report.add_failed()
+                print(colored(text=f'Error: {e}', color='red'))
+                if debug or __debug:
+                    print(colored(text=f'Traceback: {traceback.format_exc()}', color='red'))
+                continue
+            else:
```
Comment (amiwrpremium) [inline src/airdrops/set_trustline.py]: Wrong else
(The other else still has a wrong indent)

**Review by amiwrpremium — CHANGES_REQUESTED** [review]
Cahnges required

**Comment by samitabar** [comment]
Fixed The Bugs Mentioned Above

**Comment by samitabar** [comment]
Fixed The Problems Mentioned

**Comment by amiwrpremium** [comment]
Looks Good

**Pull request merged**